kong/db/schema/others/declarative_config.lua

```diff
@@ -506,6 +507,75 @@ local function validate_references(self, input)
 end
 
 
+function DeclarativeConfig.validate_references_full(self, input)
+  return validate_references(self, input)
```

Comment:
Shall we just export the function, and make `full_sync` a parameter of the function?
Say

```lua
function DeclarativeConfig:validate_references(input, full_sync)
```

Reply:
THese two functions have different parmaters, if we combine them, it will have 4 parameter. 
<img width="554" alt="image" src="https://github.com/user-attachments/assets/e12a2846-5b00-4e26-a5b9-93c27c54fb9d" />
so i think current implement is ok